plottable.d.ts

```diff
@@ -1840,8 +1840,6 @@ declare module Plottable {
         protected _scale: Scale<D, number>;
         private _formatter;
         private _orientation;
-        protected _computedWidth: number;
-        protected _computedHeight: number;
```

Comment:
This is an API break.  Instead of removing this, can we put a comment above the function stating that these variables are deprecated?